contracts/src/PDPService.sol

```diff
@@ -111,6 +112,21 @@ contract PDPService {
         return rootLeafCounts[setId][rootId];
     }
 
+    // owner proposes new owner
+    function proposeProofSetOwner(uint256 setId, address newOwner) public {
+        require(proofSetLive(setId), "Proof set not live");
+        require(proofSetOwner[setId] == msg.sender, "Only the current owner can propose a new owner");
+        proofSetProposedOwner[setId] = newOwner;
```

Comment:
I suggest giving the current owner a way to clear the proposal by proposing self as new owner (which should not require claiming).